src/components/Book/Book.js

```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from 'react';
 import './Book.css';
 import { useDispatch, useSelector } from 'react-redux';
 import { uuid } from 'uuidv4';
+import { FaCircleNotch } from 'react-icons/fa';
 import { addBook, getBooks, removeBook } from '../../redux/books/books';
 
 const Book = () => {
```

Comment:
- [ ] It happens that when I go to the other tab and come back to books, the books are fetched again and append again to the list. This is unintended behavior and the styling should not affect it. Let's fix this bug so the books are stored in redux correctly and the action doesn't get dispatched again when the component re-renders.

Tip: I think we could do some check before fetching the books again.

Reply:
@D4nielJ Thank you 🙏 for your observation. The bug has been fixed.